cliquet/resource/viewset.py

```diff
@@ -153,7 +153,9 @@ def is_endpoint_enabled(self, endpoint_type, resource_name, method,
         Uses the settings to tell so.
         """
         readonly_enabled = asbool(settings.get('readonly'))
-        if readonly_enabled and method not in self.readonly_methods:
+        readonly_method = method.lower() in [m.lower() for m in
```

Comment:
nitpick: shouldn't we normalize the other way ? e.g. the request method is always uppercase and readonly_methods could be normalize on startup